### PR TITLE
It is necessary to mention that picoTTS must be installed on the system.

### PR DESCRIPTION
**Pull request by martin3000**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/picotts.markdown [hunk 1]
```diff
@@ -11,7 +11,7 @@ ha_platforms:
 ---
 
 The `picotts` text-to-speech platform uses [Pico TTS library](https://github.com/naggety/picotts) to read out text with natural sounding voices.
-Pico TTS is a powerful open-source engine that runs locally (cloudless) so it can work even without an internet connection.
+Pico TTS is a powerful open-source engine that runs locally (cloudless) so it can work even without an internet connection. This requires to install the pico tts library on the system, typically on debian just do `sudo apt-get install libttspico-utils`.
```
Comment (frenck) [inline source/_integrations/picotts.markdown:14]: That is very specific for a specific installation type on a specific system.

Please move this into its own section specifying for which installation types this applies.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by martin3000** [comment]
What about `This requires to install the pico tts library on the system` ?

**Comment by frenck** [comment]
@martin3000 Why did you close this?

> What about This requires to install the pico tts library on the system ?

It is pre-installed on Home Assistant OS, Supervised & Container installation methods. Which is why this is confusing. One of those people might think they need to do something.

So adding a section to make clear for which users it applies would be really helpful.

**Comment by martin3000** [comment]
I don't know for which users it applies.

**Comment by frenck** [comment]
I think it is safe to say that it only applies to users running the Home Assistant Core installation method.

**Pull request closed without merging**